client/mycart/src/app/shared/footer/footer.component.html

```diff
@@ -0,0 +1,95 @@
+        <span>Get connected with us on social networks:</span>
+      </div>
+     
+      <div>
+        <a href="" class="me-4 text-reset">
+          <i class="fab fa-facebook-f"></i>
+        </a>
+        <a href="" class="me-4 text-reset">
+          <i class="fab fa-twitter"></i>
+        </a>
+        <a href="" class="me-4 text-reset">
+          <i class="fab fa-google"></i>
+        </a>
+        <a href="" class="me-4 text-reset">
+          <i class="fab fa-instagram"></i>
+        </a>
+        <a href="" class="me-4 text-reset">
+          <i class="fab fa-linkedin"></i>
+        </a>
+        <a href="" class="me-4 text-reset">
+          <i class="fab fa-github"></i>
+        </a>
+      </div>
+    </section>
+
+    <section class="">
+      <div class="container text-center text-md-start mt-5">
+        <div class="row mt-3">
+          <div class="col-md-3 col-lg-4 col-xl-3 mx-auto mb-4">
+            <h6 class="text-uppercase fw-bold mb-4">
+              <i class="fas fa-gem me-3"></i>MY CART
+            </h6>
+            <p>
+              Here you can browse, pre-order and shop for different variety of products. An awesome website that makes you want to shop more!!! 
+            </p>
+          </div>
+  
+          <div class="col-md-2 col-lg-2 col-xl-2 mx-auto mb-4">
+            <h6 class="text-uppercase fw-bold mb-4">
+              Products
+            </h6>
+            <p>
+              <a href="#!" class="text-reset">Electronics</a>
+            </p>
+            <p>
+              <a href="#!" class="text-reset">Fashion</a>
+            </p>
+            <p>
+              <a href="#!" class="text-reset">Grocery items</a>
+            </p>
+            <p>
+              <a href="#!" class="text-reset">Sports items</a>
+            </p>
+          </div>
+  
+          <div class="col-md-3 col-lg-2 col-xl-2 mx-auto mb-4">
+            <h6 class="text-uppercase fw-bold mb-4">
+              Useful links
+            </h6>
+            <p>
+              <a href="#!" class="text-reset">Register</a>
+            </p>
+            <p>
+              <a href="#!" class="text-reset">login</a>
+            </p>
+            <p>
+              <a href="#!" class="text-reset">About</a>
+            </p>
+           
+          </div>
+  
+          <div class="col-md-4 col-lg-3 col-xl-3 mx-auto mb-md-0 mb-4">
+            <h6 class="text-uppercase fw-bold mb-4">Contact</h6>
+          <p><i class="fas fa-home me-3"></i> Kochi, Ernakulam, Kerala</p>
+            <p>
+              <i class="fas fa-envelope me-3"></i>
+              team4@outlook.com
+            </p>
+            <p><i class="fas fa-phone me-3"></i> + 91 934 897 565 8</p>
+            <p><i class="fas fa-print me-3"></i> + 91 897 676 564 2</p>
+          </div>
+        </div>
+      </div>
+    </section>
+  
+    <div class="text-center p-4" style="background-color: rgba(0, 0, 0, 0.05);">
+      © 2021 Copyright:
```

Comment:
Use HTML entities form copyright symbol.

Reply:
&copy used to create copyright symbol.